cmd/buildah/commit.go

```diff
@@ -158,6 +158,10 @@ func commitCmd(c *cobra.Command, args []string, iopts commitInputOptions) error
 		}
 	}
 
+	// Add builder information.
+	builder.SetLabel("org.opencontainers.image.builder.name", "buildah")
+	builder.SetLabel("org.opencontainers.image.builder.version", buildah.Version)
```

Comment:
Please make these annotation keys constants that we export, so that API consumers can refer to them without risking typos.

Reply:
That's a decent thought, I've exported the annotation key.